### ISSUE TITLE
Can not delete page tree element with name null

### ISSUE THREAD

**Issue by WolfgangFahl** [structr/structr#529]
The remove button for this element is not accessible:
<img width="355" alt="grafik" src="https://user-images.githubusercontent.com/1336221/57773592-a99bef00-7718-11e9-80d9-000edb1625d1.png">

it disappears when the mouse is moved!

And double clicking leads to the message:

```
HTTP ERROR 404

Problem accessing /structr/html/null. Reason:

    Not Found
```

**Comment by WolfgangFahl** [comment]
The workaround is to insert an html element via the context menu - then the top element is accessible and can be removed.

**Comment by vigorouscoding** [comment]
I just tested clearing the name of a page with the most recent version, it then showed `[Page]` in the UI. If I set the name to a space character, it shows `(blank name)` so I can't test this. Can you provide a way to reproduce this behaviour?

Also, which version are you using? The latest release or the latest SNAPSHOT version?

Thanks for your help!

**Comment by WolfgangFahl** [comment]
The version is structr-3.2.1-201904081513.bb967.
I do not remember how the situation was created. At the time i had no clue how to use the Pages tab. I didn't know about the tree view and tried to delete a page via the "Properties" menu. There i clicked a "x" which was probably the "x" beside the name. After doing this the name was set to null and the behavior described above set it.

**Comment by vigorouscoding** [comment]
Ah - the key to reproducing this behaviour is reloading the page.
Upon setting the name to null, the UI reacts correctly but when refreshing the page it breaks.
I'll fix that - thanks for letting us know!

**Comment by vigorouscoding** [comment]
Fixed with 663b1e7569aa4ac769cbc40420b16174f641ec41
